Week3/homework/step2-4.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 function Dog() {
-  // add your code here
+  this.name = 'Fino';
+  this.color = 'white';
+  this.numLegs = 4;
```

Comment:
Ah. This is not completely right. You are correct in the way of creating a constructor function however, the values that you have provided to it i.e `Fino`, `white` and `4` are all hardcoded. This means, I can't use this constructor function to create a new dog object. The right way of doing it will be using function parameters:
```
function Dog(name, color, legs) {
  this.name = name;
  this.color = color;
  this.numLegs = legs;
}
```

And now you can use this constructor to create any number of `dog` objects as you like.

```
const one = new Dog('Caramel`, 'Beige', 4);
const two = new Dog('Georgie', 'Black', 4);
// and so on
```